content/en/docs/tasks/configure-pod-container/pull-image-private-registry.md

```diff
@@ -187,7 +187,7 @@ wget -O my-private-reg-pod.yaml https://k8s.io/examples/pods/private-reg-pod.yam
 In file `my-private-reg-pod.yaml`, replace `<your-private-image>` with the path to an image in a private registry such as:
 
 ```none
-janedoe/jdoe-private:v1
+yourprivateregistry.com/janedoe/jdoe-private:v1
```

Comment:
Hi @raja-anbazhagan 

I'm not clear on why this name change helps. Is it because the registry name needs to contain a `.`?
If so, how about:
```suggestion
your-private-registry.example.com/janedoe/jdoe-private:v1
```

because `example.com` is a domain name set aside for use in documentation.

Reply:
@sftim A private registry needs a server name along with its repository name. Because it is missing, the example is not working.

And also, in the same page `yourprivateregistry.com` was used as an example registry server name. So to be in sync with existing material I used the same domain name. I could clean that up for you as well in the same PR if you insist.

Reply:
Cool, that makes sense. No need to adjust the domain name, that should be a separate PR.
/lgtm